x-pack/README.md

```diff
@@ -48,69 +48,76 @@ yarn test:server
 
 #### Running functional tests
 
-The functional tests are run against a live browser, Kibana, and Elasticsearch install. They build their own version of elasticsearch and x-pack-elasticsearch, run the builds automatically, startup the kibana server, and run the tests against them.
+The functional UI tests, the API integration tests, and the SAML API integration tests are all run against a live browser, Kibana, and Elasticsearch install. Each set of tests is specified with a unique config that describes how to start the Elasticsearch server, the Kibana server, and what tests to run against them. The sets of tests that exist today are *functional UI tests* ([specified by this config](test/functional/config.js)), *API integration tests* ([specified by this config](test/api_integration/config.js)), and *SAML API integration tests* ([specified by this config](test/saml_api_integration/config.js)).
```

Comment:
Should we say something here about this requiring the `default distribution` which includes x-pack?
I'm thinking that the functional tests which are not in the x-pack directory could run against either the default distribution or the OSS dist?

Reply:
Ah, the OSS functional tests are only meant to run against OSS kibana. That's my understanding. I don't know if they're meant to be run against x-pack specifically, but they currently work if you have the x-pack directory present (but you're running from the root kibana directory).

Reply:
That brings up a good point that I'm not sure is clear in the READMEs.  When you run tests from the kibana folder, does it start the OSS build or the default build?
Obviously when you start it from the x-pack folder it would have to start the default distribution.

Sorry if this is explained somewhere and I just didn't see it yet.

Reply:
So if I wanted to run *all* functional tests without having to restart Elasticsearch or Kibana could I;
```
cd x-pack
node scripts/functional_tests_server
cd ..
node scripts/functional_test_runner --config test/functional/config.js,x-pack/test/functional/config.js
```

I think by going into the x-pack directory and starting functional_tests_server I'm getting Elasticsearch and Kibana started up with all features.
And then from kibana root directory I could run the kibana functional tests and the x-pack functional tests?